modules/rosa-cluster-autoscaler-ui-settings.adoc

```diff
@@ -46,30 +68,35 @@ The tables explain all the configurable UI settings when using cluster autoscali
 |Type or Range
 |Default
 
-|`maxNodesTotal`
-|Maximum number of nodes in all node groups. Includes all nodes, not just automatically scaled nodes. The cluster autoscaler does not grow the cluster beyond this number.
-|`integer`
-|0
-
-|`cores.min`
+|`cores-total-min`
 |Minimum number of cores in cluster. The cluster autoscaler does not scale the cluster beyond these numbers.
 |`object`
 |1
 
-|`cores.max`
+|`cores-total-max`
 |Maximum number of cores in cluster. The cluster autoscaler does not scale the cluster beyond these numbers.
 |`object`
 |320000
 
-|`memory.min`
+|`memory-total-min`
 |Minimum number of gigabytes of memory in cluster. The cluster autoscaler does not scale the cluster beyond these numbers.
 |`object`
 |1
 
-|`memory.max`
+|`memory-total-max`
 |Maximum number of gigabytes of memory in cluster. The cluster autoscaler does not scale the cluster beyond these numbers.
 |`object`
 |6400000
+
+|`max-nodes-total`
+|Maximum number of nodes in all node groups. Includes all nodes, not just automatically scaled nodes. The cluster autoscaler does not grow the cluster beyond this number.
+|`integer`
+|0
+
+|GPUs
+|Minimum and maximum number of different GPUs in cluster, in the format <gpu_type>:<min>:<max>. Cluster autoscaler does not scale the cluster beyond these numbers. Can be passed multiple times.
```

Comment:
the part of ``Can be passed multiple times`` doesn't seem related to the UI usage
@celdrake wdyt?

Reply:
Agree. In the UI, this is a comma separated list, in which each item in the list is in the format `<gpu_type>:<min>:<max>`